compiler/packages/babel-plugin-react-compiler/src/HIR/BuildHIR.ts

```diff
@@ -2130,24 +2130,41 @@ function lowerExpression(
             suggestions: null,
           });
         }
-        const fbtEnumLocations: Array<SourceLocation> = [];
+        // see `error.todo-multiple-fbt-plural` fixture for explanation
+        const fbtLocations = {
+          enum: new Array<SourceLocation>(),
+          plural: new Array<SourceLocation>(),
+          pronoun: new Array<SourceLocation>(),
+        };
```

Comment:
These can be tracked separately as fbt does not dedupe across enum/plural/pronouns